hack/mirror-origin-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+. $(dirname "$0")/origin-tests-common.sh
+
+skopeo_workdir=$(mktemp -d)
+mapping_file=$(mktemp)
+function finish {
+  rm -rf ${skopeo_workdir}
+  rm -f ${mapping_file}
+}
+trap finish EXIT
+
+MIRROR_ORIGIN_TESTS_OC_TOOL="${MIRROR_ORIGIN_TESTS_OC_TOOL:-oc}"
+MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI="${MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI:-docker}"
```

Comment:
same goes for this one. it's illogical to set docker for one script and podman (or other) for others.